src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/AlgorithmOperations.java

```diff
@@ -209,7 +209,6 @@ public AlgorithmOperations(DSLContext create) {
                 .leftJoin(CHOSEN_TASK_CHOOSER_PARAM).onKey()
                 .where(CHOSEN_TASK_CHOOSER_PARAM.EXPERIMENT.eq(experimentID))
                 .and(ALGORITHM_TASK_CHOOSER_PARAM.ALGORITHM.eq(taskChooserID))
-                .groupBy(ALGORITHM_TASK_CHOOSER_PARAM.fields())
```

Comment:
uh, i committed them on the wrong branch sry. They are bug-fixes for the db-package